CONTRIBUTING.md

```diff
@@ -145,6 +145,12 @@ test out the patch set and opine on the technical merits of the patch. Please,
 first review PR on the conceptual level before focusing on code style or
 grammar fixes.
 
+### API changes
+
+All PRs that change the public API of `rust-bitcoin` must include a patch to the
```

Comment:
In 0f28f33bfd9fdb231e814d81a16772a286b21e28:

Should clarify that this applies only to `io`, `hashes`, `primitives` and `units`.